idl/chromadb/proto/chroma.proto

```diff
@@ -102,6 +102,8 @@ service MetadataReader {
 
 message CountRecordsRequest {
     string segment_id = 1;
+    // required
+    optional string collection_id = 2;
```

Comment:
Why is this optional?

Reply:
without optional, we can't distinguish between a default and a user-provided value
although we're not currently doing this, so I guess we could drop the optional

Reply:
I think we should drop the optional then?

Reply:
sure, we can also drop it on the other requests